SetupPrivateJupyterLab.sh

```diff
@@ -18,14 +18,16 @@ if [ "$OWNER" != "" ] && [ "$CONNECT_GROUP" != "" ]; then
     cd /home/$OWNER
 
     chown -R  $OWNER: /venv
-    
+    # Install https://github.com/astral-sh/uv for users editing environment later
+    curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
@ivukotic is it correct that if we want something to be accessible to the final end user (and in this case also modify their `~/.bash_profile` that all of that code needs to exist in `SetupPrivateJupyterLab.sh`? I believe this is the case, but I forget how this all actually works at request time.